conda_smithy/cli.py

```diff
@@ -184,6 +184,8 @@ def __call__(self, args):
         try:
             configure_feedstock.main(args.feedstock_directory)
             print("\nCI support files regenerated. These need to be pushed to github!")
+            print("\nYou can commit the changes with:\n\n"
+                  "    git commit -am 'MNT: rerender with conda-smithy %s'" % __version__)
```

Comment:
With a '1.2.0' version, it would be 37 characters. Adding 'version' would make it 45 (can save 5 by removing `MNT: `).

I can easily remove the commit command, if it's more trouble than it's worth.

The main thing I'm after is something pasteable into a commit message, since I find myself looking up the conda-smithy version with `conda-smithy --version` every time I rerender a feedstock.

Even just printing:

    'Rerendered with conda-smithy <version>'

at the end would scratch my itch right now.